perfkitbenchmarker/openstack/os_virtual_machine.py

```diff
@@ -75,10 +75,16 @@ def _Create(self):
         image_id = image.id
         boot_from_vol = []
         if FLAGS.openstack_boot_from_volume:
+
+            if FLAGS.openstack_volume_size > 0:
+                volume_size = FLAGS.openstack_volume_size
+            else:
+                volume_size = flavor.disk
+
```

Comment:
I did not look through the rest of the code but it might be worth logging the size chosen here.